ksqldb-engine/src/main/java/io/confluent/ksql/ddl/commands/DropTypeFactory.java

```diff
@@ -15,14 +15,32 @@
 
 package io.confluent.ksql.ddl.commands;
 
+import com.google.common.annotations.VisibleForTesting;
 import io.confluent.ksql.execution.ddl.commands.DropTypeCommand;
+import io.confluent.ksql.metastore.MetaStore;
 import io.confluent.ksql.parser.DropType;
+import io.confluent.ksql.util.KsqlException;
+
+import java.util.Objects;
 
 public class DropTypeFactory {
-  DropTypeFactory() {
+  private final MetaStore metaStore;
+
+  @VisibleForTesting
```

Comment:
nit: not visible for testing only. Used by production code,, right?

Reply:
Right, removed.